doc/app-example/CMakeLists.txt

```diff
@@ -1,23 +1,28 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
+# Warning: This example assumes that you are building on a host
+# with pkg-config available (e.g. linux). The logic required to
+# build under windows/mingw and/or mac was intentionally omitted
+# to keep this CMakeLists as small as possible
 
-PROJECT(st-hello)
-SET(PROJECT_VERSION   0.1)
-SET(SRCS main.c)
+cmake_minimum_required(VERSION 2.8)
```

Comment:
> cmake_minimum_required should always be the first line for CMakeLists.txt

It is there on the first line, diff just looks a bit weird due to adding sidenotes.